yaas/bootstrap.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+
+
+class Bootstrap:
+
+    """ Interact with agent bootstrap requests. """
+
+    def __init__(self, client):
+        self.client = client
+
+    def agents(self, hosts, sshKey, user, userRunAs=None):
+        """ Install and configure ambari-agent on hosts. """
+        req = {
+            'hosts': hosts,
+            'sshKey': sshKey,
+            'user': user,
+            }
+        if userRunAs is not None:
+            req['userRunAs'] = userRunAs
+        response = self.client.request(
+            'post',
+            '/api/v1/bootstrap',
+            json=req)
+        return response.json()
```

Comment:
Yes, so you're right, yaas should strip out the hrefs (i.e. pick out the valuable not href stuff to print - tomato tomato). If I'm importing yaas, though, to interact with Ambari, I want the whole JSON response back so I can do with it what I please because I'm a badass library that does badass potentially href-y things.